### PR TITLE
DE37821 calcs suck

### PR DESCRIPTION
**Pull request by ChrisKraljevicD2L**
Turns out fixing the CSS for the sidenav involves changing stuff over here

### CODE REVIEW

File: d2l-sequence-navigator/d2l-sequence-navigator.js [hunk 1]
```diff
@@ -43,7 +43,7 @@ PolymerElement
 		}
 
 		.module-content {
-			height: calc( 100% - 259px );
+			height: calc( 100% - 203px );
```
Comment (ianlaidlaw) [inline d2l-sequence-navigator/d2l-sequence-navigator.js]: why 203? does this number have any real significance?

Comment (ChrisKraljevicD2L) [reply d2l-sequence-navigator/d2l-sequence-navigator.js]: I removed the 56px `top` from the sidebar, so this is 259 - 56.

Comment (ianlaidlaw) [reply d2l-sequence-navigator/d2l-sequence-navigator.js]: sucks that this component has to be so aware of the parent using it.. I know that the launcher FRA uses this component, will shrinking this be an issue?

Comment (ChrisKraljevicD2L) [reply d2l-sequence-navigator/d2l-sequence-navigator.js]: I will test this out in sequence launcher. I have a sinking feeling that it hasn't been updated in a very long time.

**Pull request merged**